source/includes/webhooks.md

```diff
@@ -0,0 +1,97 @@
+# Webhooks
+
+Highline’s daily operations involve a number of asynchronous events regarding payments, data, and other subsets of our product. These can range from updating the status of a customer’s employment to settling funds with clients. 
+
+In order to maintain clarity and consistency with our partners, Highline uses webhooks to communicate key events regarding the bill payment process. This allows clients to receive notifications anytime a relevant event is impacted by our operations.
+
+## Setup
+Custom webhook endpoints can be created from the [company settings page](https://app.highline.co/profile). Navigate to the client portal and create a new webhook and start receiving following webhook events
+
+## Bill Payment Lifecycle
+The following events relate to the entirety of Highline’s bill payment lifecycle. 
+
+
+## Topics
+
+### Bill Payments
+
+> <div class="code-block-title">Example Payload</div>
+
+```json
+  {
+    "topic": "bill_payments",
+    "event": "paid",
+    "metadata": {
+      "product_id": "24de3..."
+    },
+    "data": {
+      "bill_payment_id": "da345..."
+    }
```

Comment:
Why are we breaking metadata and data. Can we package all in data?

Reply:
This was the initial design I met, and It's being used for processing argyle webhooks, hence I decided to reuse it.

Reply:
Argyle is not a good example, MT and others are better. In this case, we sent the payload in a flat format and data and metadata both are not required.